### PR TITLE
fix(authoring): fix duplicate authoring header cvs

### PR DESCRIPTION
**Pull request by petrjasek**
add function to get all custom header cvs for profile editor

SDNTB-550

reverts https://github.com/superdesk/superdesk-client-core/pull/2766

### CODE REVIEW

File: scripts/apps/authoring/metadata/metadata.js [hunk 1]
```diff
@@ -1199,16 +1199,21 @@ function MetadataService(api, subscribersService, config, vocabularies, $rootSco
...
             }));
         },
         getCustomVocabulariesForArticleHeader: function(qcodes, editor, schema) {
             return this.getFilteredCustomVocabularies(qcodes)
                 .then(
-                    (cvs) => cvs.filter((cv) => cv.items && cv.items.length && (editor[cv._id] || schema[cv._id]))
+                    (cvs) => cvs.filter((cv) => cv.terms.length && (editor[cv._id] || schema[cv._id]))
                 );
         },
+        getAllCustomVocabulariesForArticleHeader: function(editor, schema) {
```
Comment (tomaskikutis) [inline scripts/apps/authoring/metadata/metadata.js]: Could you add a comment on how `getAllCustomVocabulariesForArticleHeader` and `getCustomVocabulariesForArticleHeader` differ and where are they used?

**Comment by petrjasek** [comment]
@tomaskikutis this is it

File: scripts/apps/authoring/metadata/metadata.js (hunk 1 above)
Comment (tomaskikutis) [reply scripts/apps/authoring/metadata/metadata.js]: What was the problem for ntb, why weren't the content profiles working?

Comment (petrjasek) [reply scripts/apps/authoring/metadata/metadata.js]: that's not actually fixing those content profiles not working, it's only fixing issue with some cvs displayed in content tab

Comment (tomaskikutis) [reply scripts/apps/authoring/metadata/metadata.js]: What was then the issue with cvs in the content tab?

**Comment by tomaskikutis** [comment]
I tried to run this PR with ntb master API and the situation with content profiles seems to be the same - I can't add anything and see an error from the API.

500 api/content_types/59105a86177e5b00e602fe2c?edit=true

**Comment by petrjasek** [comment]
that's right, I didn't push a fix for that yet, it's a problem with init data

**Pull request merged**